cms/src/schemas/constants.js

```diff
@@ -921,7 +932,7 @@ export const clinicalTumorDiagnosisDependent = {
       'Stage IVA',
       'Stage IVB',
     ],
-    'rare cancers': [],
+    'rare cancers': ['Stage IV', 'Stage 0', 'Stage I', 'Stage II', 'Stage III'],
```

Comment:
Is the weird ordering here intentional?

Reply:
copied from eva - was being lazy and didn't reorder